onchain_chainalysis.go

```diff
@@ -0,0 +1,40 @@
+package chainalysis
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/0xsequence/ethkit/ethrpc"
+)
+
+type onChainalysis struct {
```

Comment:
I think to avoid using this thing at all.. lets just completely remove this please, it's super inefficient, and should always be using an index instead